### PR TITLE
database entries added + some changes to boss

### PR DESCRIPTION
**Pull request by elthehablo**
hope this is better now. lemme know what still needs to be changed if it doesn't work.

### CODE REVIEW

**Comment by elthehablo** [comment]
just noticed a mistake. gonna fix that

File: database/world/customWorldboss-down.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
 DELETE FROM `creature_template` WHERE `entry` IN 
-(1112001,1112002,1112003,1112011,1112012,1112013,1112021,1112022,1112023,1112031,1112032,1112033,1112041,1112042,1112043,1112051,1112052,1112053,1112999);
-DELETE FROM `npc_text` WHERE `ID` IN (91101,91102,91111,91112,91113,91114,91115,91116);
-DELETE FROM `gossip_menu` WHERE `MenuID` IN (62001,62002,62003,62004,62005,62006);
+(1112001,1112002,1112003,1112011,1112012,1112013,1112021,1112022,1112023,1112031,1112032,1112033,1112041,1112042,1112043,1112051,1112052,1112053,1112061,1112062,1112063,1112999);
+DELETE FROM `npc_text` WHERE `ID` IN (91101,91102,91111,91112,91113,91114,91115,91116),91117;
```
Comment (elthehablo) [inline database/world/customWorldboss-down.sql:3]: something went wrong here.. oops

**Comment by elthehablo** [comment]
should work now. @55Honey pls review

File: customWorldboss.lua [hunk 2]
```diff
@@ -731,9 +732,97 @@ Config_bossYellPhase2[7] = "Sssssszzzzzzzzz!"
...
+Config_bossSpellModifier7bp0[8] = nil      -- not required if nil
+Config_bossSpellModifier7bp1[8] = nil      -- not required if nil
+Config_bossSpellModifier8bp0[8] = nil      -- not required if nil
+Config_bossSpellModifier8bp1[8] = nil      -- not required if nil
+
+Config_addHealthModifierParty[8] = 0.2     -- modifier to change health for party encounter. Value in the SQL applies for raid
+Config_addsAmount[8] = 6                   -- how many adds will spawn
+
+Config_addSpell1[8] = 38085             -- min range 30m, 1-3rd farthest target within 30m -- Shadow Blast (just damage)
+Config_addSpell2[8] = 36541              -- min range 45m, cast on tank -- Curse of Burning Shadows (deals shadow damage when fire damage is taken)
```
Comment (55Honey) [inline customWorldboss.lua:790]: OCD is kicking in.
```suggestion
Config_addSpell2[8] = 36541             -- min range 45m, cast on tank -- Curse of Burning Shadows (deals shadow damage when fire damage is taken)
```

File: customWorldboss.lua [hunk 3]
```diff
@@ -731,9 +732,97 @@ Config_bossYellPhase2[7] = "Sssssszzzzzzzzz!"
...
+
+Config_bossSpellTimer1[8] = 10000       -- This timer applies to Config_bossSpell1
+Config_bossSpellTimer2[8] = 37000       -- This timer applies to Config_bossSpell2
+Config_bossSpellTimer3[8] = 12000       -- This timer applies to Config_bossSpellSelf in phase 1 and Config_bossSpell3+4 randomly later
+Config_bossSpellTimer5[8] = 20000       -- This timer applies to Config_bossSpell5+6
+Config_bossSpellTimer7[8] = 40000       -- This timer applies to Config_bossSpell7+8 (in ms)
+Config_bossSpellEnrageTimer[8] = 90000
+Config_minPhaseForTimer7[8] = 3         -- From this phase on the boss will use Timer 7 to cast Spell 7+8. Will ignore it before.
+
+Config_bossSpellModifier1bp0[8] = 5000     -- should be instant damage ??? @55Honey
```
Comment (55Honey) [inline customWorldboss.lua:769]: Yes, I think so, too. These values are before effects though. So they need to be tested, e.g. on the PTR.

File: database/world/customWorldboss-down.sql [hunk 4]
```diff
@@ -1,7 +1,7 @@
 DELETE FROM `creature_template` WHERE `entry` IN 
-(1112001,1112002,1112003,1112011,1112012,1112013,1112021,1112022,1112023,1112031,1112032,1112033,1112041,1112042,1112043,1112051,1112052,1112053,1112999);
-DELETE FROM `npc_text` WHERE `ID` IN (91101,91102,91111,91112,91113,91114,91115,91116);
-DELETE FROM `gossip_menu` WHERE `MenuID` IN (62001,62002,62003,62004,62005,62006);
+(1112001,1112002,1112003,1112011,1112012,1112013,1112021,1112022,1112023,1112031,1112032,1112033,1112041,1112042,1112043,1112051,1112052,1112053,1112061,1112062,1112063,1112999);
```
Comment (55Honey) [inline database/world/customWorldboss-down.sql:2]: You've added the 60s values for the encounter 7 here. Seems these were forgotten in the down query last time. Wanna add 70s for encounter 8 as well? Same for line 3, 4 and 7. 😃

**Review by 55Honey — CHANGES_REQUESTED** [review]
Thanks for the PR! Excellent performance!
Let's address these minor amendments and throw it on the PTR?

File: database/world/customWorldboss-down.sql (hunk 4 above)
Comment (elthehablo) [reply database/world/customWorldboss-down.sql:2]: lol yeah I got messed up with the numbers in there. I will update for my encounter

File: customWorldboss.lua (hunk 3 above)
Comment (elthehablo) [reply customWorldboss.lua:769]: yeah I'm down to do that if you're down to bring the npc back on the PTR

Comment (55Honey) [reply customWorldboss.lua:769]: The "Amber Haze" NPC near Gadgetzan can only spawn 5man encounters, hence a single add.
But I'll happily spawn a Chromie so everyone can try it.

**Comment by elthehablo** [comment]
should be good now

File: customWorldboss.lua [hunk 5]
```diff
@@ -731,9 +732,97 @@ Config_bossYellPhase2[7] = "Sssssszzzzzzzzz!"
...
+Config_addSpellModifier3bp1[8] = nil    -- not required if nil
+
+Config_aura1Add1[8] = nil               -- an aura to add to the 1st add
+Config_aura2Add1[8] = nil               -- another aura to add to the 1st ad
+Config_aura1Add2[8] = nil               -- an aura to add to the 2nd add
+Config_aura2Add2[8] = nil               -- another aura to add to the 2nd add
+Config_aura1Add3[8] = nil               -- an aura to add to all ads from the 3rd on
+Config_aura2Add3[8] = nil               -- another aura to add to all add from the 3rd on
+
+Config_addSpell4Yell[8] = "Bathe in the shadows... master!"            -- yell for the adds when Spell 4 is cast
```
Comment (55Honey) [inline customWorldboss.lua:814]:
```suggestion
Config_addSpell3Yell[8] = "Bathe in the shadows... master!"            -- yell for the adds when Spell 3 is cast
```
There's no system to send a yell for spell 4. We need to stick with spell 3 for this one.

**Review by 55Honey — CHANGES_REQUESTED** [review]

File: customWorldboss.lua (hunk 3 above)
Comment (elthehablo) [reply customWorldboss.lua:769]: yeah that would be nice :)

**Comment by elthehablo** [comment]
fixed requested changes

**Comment by elthehablo** [comment]
fight should be in working order and ready to be tested now. might still need some nerfing

**Comment by 55Honey** [comment]
@elthehablo what's the state of this PR?

**Comment by elthehablo** [comment]
> @elthehablo what's the state of this PR?

Pending due to lack of group testers. I need testing to tune

**Pull request closed without merging**